elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```diff
@@ -559,7 +559,7 @@ pfe-navigation-item.expanded pfe-navigation-item[parent_hidden]:not(.expanded) [
 
 pfe-navigation-main [slot="trigger"],
 pfe-navigation-main [slot="trigger"] > a {
-    font-size: $trigger--FontSize--fallback;
+    font-size: #{pfe-local($region: trigger, $cssvar: FontSize, $fallback: $trigger--FontSize--fallback)};
```

Comment:
This variable `$trigger--FontSize--fallback` is equal to: `pfe-var(font-size--heading--epsilon)` which is the design system specified size for the trigger link for navigation at this breakpoint.

Reply:
I think the challenge here is that these styles are not browser-scoped and maybe they should be? `@include browser-query(ie11 edge) {` <- @chrisdo1 Do you forsee any issue in wrapping these styles in a browser query or is there another way we can scope these so that the hardcoded values don't prevent the variables from working in modern experiences?

Reply:
@starryeyez024 ^ here's where I'm seeing the variable that references epsilon but looking at the code, it's very widely scoped and I think maybe we need some browser query encapsulation here.